js/components/report/iframe-answer.js

```diff
@@ -36,22 +32,47 @@ export default class IframeAnswer extends PureComponent {
     */
     try {
       const json = JSON.parse(answer);
-      const interactiveState = JSON.parse(json.interactiveState);
-      return interactiveState.lara_options.reporting_url;
+      const interactiveState = json.interactiveState ? JSON.parse(json.interactiveState) : null;
+      if (interactiveState && interactiveState.lara_options && interactiveState.lara_options.reporting_url) {
+        return interactiveState.lara_options.reporting_url;
+      }
+      return null;
     } catch (e) {
       return answer;
     }
   }
 
+  /**
+   * Adds a studentId and iframeQuestionId to the existing url
+   */
+  getStandaloneLinkUrl(question, answer) {
+    const baseUrl = `${window.location.origin}${window.location.pathname}`;
```

Comment:
I'm not sure what the current use-case would be. Unless we start pushing state into the browser history as the teacher clicks around. But then we wouldn't necessarily want that maintained when viewing the standalone iframe.

It is possible, it's true, but there would already be a number of places that url location code would need to be updated once we did that, and we'd probably need to consider each individually instead of being surprised.